meta-lxatac-software/recipes-rust/tacd/files/tacd-failsafe.sh

```diff
@@ -3,5 +3,6 @@
 # Disable DUT power and enable discharge resistor (both lines are active low).
 # gpiofind returns a chip + id pair in the form of: gpiochip0 2.
```

Comment:
Please update the comment as well.

Also describe why it's useful to keep the gpioset running.

Reply:
Heh. I completely missed the comment when I changed the code.
I've now written a (bit lengthy) comment about the hows and whys of `tacd-failsafe.sh`.